src/Symfony/Component/Form/Extension/Core/Type/CurrencyType.php

```diff
@@ -27,6 +29,10 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
             'choice_loader' => function (Options $options) {
+                if (!class_exists(Intl::class)) {
+                    throw new LogicException(sprintf('The "symfony/intl" component is required to use "%s".', static::class));
```

Comment:
to all messages: `Try running "composer require symfony/intl".`

Reply:
Thank you